capplets/common/capplet-util.h

```diff
@@ -33,6 +33,7 @@
 /* Retrieve a widget from the UI object */
 
 #define WID(s) GTK_WIDGET (gtk_builder_get_object (dialog, s))
+#define GET_WIDGET(x) (GTK_WIDGET (gtk_builder_get_object (builder, (x))))
```

Comment:
Move `#define GET_WIDGET...` at the top of:
- capplets/display/xrandr-capplet.c
- capplets/keybindings/mate-keybinding-properties.c
- capplets/network/mate-network-properties.c